NSeleneTests/Integration/SharedDriver/SeleneCollection_Enumerable_Should.cs

```diff
@@ -0,0 +1,40 @@
+﻿using NSelene;
+using NSelene.Tests.Integration.SharedDriver.Harness;
+using static NSelene.Selene;
+using NUnit.Framework;
+using System.Linq;
+
+namespace NSeleneTests.Integration.SharedDriver.SeleneCollectionSpec
+{
+    [TestFixture]
+    public class SeleneCollection_Enumerable_Should : BaseTest
+    {
+
+        [TearDown]
+        public void TeardownTest()
+        {
+            Configuration.Timeout = 4;
+        }
+
+        [Test]
+        public void SeleneCollection_Enumerable_ShouldReturnFirst()
+        {
+            Given.OpenedEmptyPage();
+            SeleneCollection elements = SS("#will-exist>input");
+
+            When.WithBody(@"
+                <p id='will-exist'>
+                    <input id='ask' type='submit' value='How r u?' style='display:none'></input>
+                    <input id='answer1' type='submit' value='Good!'></input>
+                    <input id='answer2' type='submit' value='Great!'></input>
+                </p>"
+            );
```

Comment:
probably no need to break code into OpenedEmptyPage and then When.WithBody
there should be something like:

Given.PageWithBody(....) that does 2 things at once

Reply:
Done